homeassistant/components/intent_script/__init__.py

```diff
@@ -148,6 +148,13 @@ async def async_handle(self, intent_obj: intent.Intent):
 
         response = intent_obj.create_response()
 
+        slots = slots | {
+            "context_items": {
```

Comment:
I am unsure about the naming of this object, feedback is welcome

Reply:
This change doesn't seem part of the current PR. Let's move this to another PR.

Reply:
True. My reasoning was:
- PRs get reviewed/merged very slowly
- the actual satellite which made the call is useful to have in responses, especially for custom sentences/custom intents

If you feel it's too much, I'm OK with taking it out.